config/kafka.js

```
@@ -93,6 +96,11 @@ var _sendToKafkaConsumers = function (topic, host) {
 			if (message && message.topic === USER_DELETE_TOPIC) {
 				userDeleteConsumer.messageReceived(message)
 			}
+
+			// call userCourses consumer
+			if (message && message.topic === COURSES_TOPIC) {
+				userCoursesConsumer.messageReceived(message)
```

ESLint

[error] 'userCoursesConsumer' is not defined.

(no-undef)